sdk/src/test/java/com/mapbox/maps/StyleTest.kt

```diff
@@ -246,6 +246,28 @@ class StyleTest {
     verify { nativeMap.invalidateStyleCustomGeometrySourceTile("id", tileId) }
   }
 
+  @Test
+  fun addPersistentStyleLayer() {
+    val value = mockk<Value>()
+    val position = mockk<LayerPosition>()
+    style.addPersistentStyleLayer(value, position)
+    verify { nativeMap.addPersistentStyleLayer(value, position) }
+  }
+
+  @Test
+  fun addPersistentStyleCustomLayer() {
+    val layerHost = mockk<CustomLayerHost>()
+    val layerPosition = mockk<LayerPosition>()
+    style.addPersistentStyleCustomLayer("id", layerHost, layerPosition)
+    verify { nativeMap.addPersistentStyleCustomLayer("id", layerHost, layerPosition) }
+  }
+
+  @Test
+  fun isStyleLayerPersistent() {
+    style.isStyleLayerPersistent("id")
+    verify { nativeMap.isStyleLayerPersistent("foobar") }
```

Comment:
Why are we checking for persistance with `id` above but verifying it with `foobar`? imho this test should fail 😄

Reply:
🤦 will fix it